app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -35,9 +45,8 @@ def view_toolbar_filename
       'dashboard_summary_toggle_view_tb'
     elsif %w[container_project].include?(@layout)
       'container_project_view_tb'
-    elsif !%w[all_tasks timeline diagnostics my_tasks miq_server usage].include?(@layout) &&
-          !@layout.starts_with?("miq_request") && @display == "main" &&
-          @showtype == "main" && !@in_a_form
+    elsif !NO_SUMMARY_VIEW_TB.include?(@layout) && !@layout.starts_with?("miq_request") &&
```

Comment:
Do we still need this change after the `toolbar` method call?

Reply:
This part of code we still need, but i will remove the specific cleanup change made in this PR since that is no longer related to chargeback rates, as we are using `toolbar` for chargeback rates.